include/net/ieee802154_radio.h

```diff
@@ -96,6 +131,11 @@ struct ieee802154_radio_api {
 	/** Stop the device */
 	int (*stop)(struct device *dev);
 
+	/** Set specific radio driver configuration. */
+	int (*set_config)(struct device *dev,
```

Comment:
since we don't have a get_config, let's call it 'config'

Reply:
Hmm, it is unnecessary hassle if we need to rename it later when we need set/get config functions. So it might be better to call this one set_config anyway.

Reply:
Done

Reply:
@jukkar on the contrary to ethernet, 15.4 devices are way much simpler and all is controlled by the L2, so there won't be any point to get the config at any time.

Reply:
That is true, but still it is more logical if we are setting things, the function name reflects the intention.

Reply:
could be "configure" then, it does reflects pretty well the intention

Reply:
So do we have a conclusion here? Stick to `config` or revert to `set_config`? Or `configure`?

Reply:
configure

Reply:
Ok, done.